src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -45,6 +45,19 @@ protected XMLCall(URI uri, String contentType) throws IOException {
         this.outcall.setRequestProperty("Content-Type", contentType);
     }
 
+    /**
+     * Creates a new XMLCall for the given URI and Content-Type.
+     *
+     * @param uri         the target URI to call
+     * @param blacklistId the id for the blacklist to use.
+     * @param contentType the value of the Content-Type property for requests
+     * @throws IOException in case the URI is blacklisted
+     */
+    protected XMLCall(URI uri, String blacklistId, String contentType) throws IOException {
```

Comment:
Who's calling this protected constructor? I think a simple Setter in Outcall would be easiest and clearest as there is no reason why the blacklist id has to be specified when constructing the object.

Reply:
The protected Constructor is overwritten with a public one in my case. I forgott to implement the methods like the XMLCall.to(...) method.
The check if a call is blacklisted happens in the Outcall constructor. A simple setter in the Outcall class would check the hostname first and then use the blacklistid later in the call.